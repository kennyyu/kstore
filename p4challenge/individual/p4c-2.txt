insert(ta,412234,tb,123231,tc,54323,td,764543)
insert(ta,1312312,tb,5321234,tc,3453241,td,14232342)
insert(ta,39485394,tb,123124,tc,23429,td,1293)
insert(ta,234231,tb,1000,tc,23425,td,21234)
insert(ta,30293,tb,123901,tc,23491,td,12303)
tb_select=select(tb,1000)
ta_fetch=fetch(ta,tb_select)
ud_select=select(ud,9000,9000000)
ua_fetch=fetch(ua,ud_select)
t_id,u_id=hashjoin(ta_fetch,ua_fetch)
td_fetch=fetch(td,t_id)
td_avg=avg(td_fetch)
uc_fetch=fetch(uc,u_id)
uc_avg=avg(uc_fetch)
tuple(td_avg,uc_avg)
