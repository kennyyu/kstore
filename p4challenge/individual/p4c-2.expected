(536077,507767)
